### PR TITLE
Add guidance for decorative images

### PR DESCRIPTION
**Pull request by AshGDS**
Hi,

Here is the added guidance regarding decorative images.

Related to issue 22: https://github.com/nationalarchives/front-end-development-guide/issues/22

Thanks

### CODE REVIEW

File: development-guide.md [hunk 1]
```diff
@@ -211,6 +211,13 @@ Where SVGs are used, they are used accessibly
 
 ### Images
 
+Decorative images should be implemented using the CSS `background-image` property, so that Assistive Technology can ignore the images. If a decorative image must be implemented using the HTML `<img>` element, you must ensure that:
```
Comment (gtvj) [inline development-guide.md]: Hi @AshTNA - this looks good. I'd de-capitalise 'Assistive Technology' and change it to 'assistive technolog**ies**'

File: development-guide.md [hunk 2]
```diff
@@ -211,6 +211,13 @@ Where SVGs are used, they are used accessibly
...
+ 
+ - The element's `title` attribute is either absent or empty. 
+ - The element's `alt` attribute is empty (`alt="""`).
```
Comment (gtvj) [inline development-guide.md]: Thanks @AshTNA - what do you think about including `role="presentation"` as an additional bullet point? It's not as well supported as the null `alt=""`, but that situation is likely to improve and it's a more declarative way of indicating (to developers too) that it's a decorative image. I'd also consider including a link to this page: https://www.w3.org/WAI/tutorials/images/decorative/

**Review by gtvj — CHANGES_REQUESTED** [review]
Hi @AshTNA - I've assigned this to myself and suggested a couple of minor changes. If you could make these I'd be happy to merge.

File: development-guide.md (hunk 2 above)
Comment (AshGDS) [reply development-guide.md]: Good idea @gtvj - I have added both of these to the development guide.

File: development-guide.md (hunk 1 above)
Comment (AshGDS) [reply development-guide.md]: Thanks @gtvj -  I have made this change.

**Review by gtvj — APPROVED** [review]
Brilliant - thanks @AshTNA 👍

**Pull request merged**